ui/app/AppLayouts/Profile/views/NotificationsView.qml

```diff
@@ -311,6 +311,7 @@ SettingsContentBase {
         }
 
         StatusListItem {
+            visible: false
```

Comment:
WDYT if you add a comment here to say we are hiding it for **2.30** since it's not mature enough?